gym/lib/gym/options.rb

```diff
@@ -103,6 +103,16 @@ def self.plain_options
                                          UI.important "Using legacy build system - waiting for radar to be fixed: https://openradar.appspot.com/radar?id=4952000420642816"
                                        end
                                      end),
+       FastlaneCore::ConfigItem.new(key: :use_generic_archive_fix,
+                                    env_name: "GYM_USE_GENERIC_ARCHIVE_FIX",
+                                    description: "Use fix for generic arhives https://github.com/CocoaPods/CocoaPods/issues/4178",
```

Comment:
There is a spelling mistake `arhives`, could you submit a PR fixing it, or was it already fixed?